### DISCUSSION TITLE
Resetting the window size for all windows in a workspace

### DISCUSSION THREAD

**Discussion by onlyspxctre** [glzr-io/glazewm#417] [Ideas]
Sometimes it is possible to accidentally resize a window, which requires a few drags to resize the window back to a comfortable size.

An idea for a remedy to the hassle:

A command that can be set to a keybind to either revert the window size state to the previous state before the accidental resize

OR

A command that reverts the window sizes to the default sizes of the windows of that particular workspace (or just run the methods that resize the windows automatically during init)

**Comment by aaronedev** [comment]
@spxctreofficial yup i agree its kind of annoying i am looking for something similar - maybe a good script that handles window positions and sizes in harmony with glazewm?
